src/app/map/page.tsx

```
@@ -3,12 +3,19 @@ import MapTabs from '@/components/map/MapTabs';
 function page() {
   return (
     <>
       <div className={flex({ direction: 'row', p: 'xs', marginB: 'sm' })}>
         {MAP_CATEGORY.map(category => (
-          <div className={flex({ width: 'auto', align: 'center' })}>
+          <div
+            className={cx(
+              flex({ width: 'auto', align: 'center' }),
+              css({ cursor: 'pointer' }),
+            )}
+          >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)